crates/next-custom-transforms/tests/fixture.rs

```diff
@@ -352,7 +355,10 @@ fn react_server_components_server_graph_fixture(input: PathBuf) {
         },
         &input,
         &output,
-        Default::default(),
+        FixtureTestConfig {
+            module: Some(true),
```

Comment:
cc @shuding FYI, now fixture tests parse tests as `Program` by default - Module if there's an import or an export, Script if not. But passes in next.js is only about modules, so I added this config.